doc/api/http2.md

```diff
@@ -1036,6 +1042,35 @@ Provides miscellaneous information about the current state of the
 
 A current state of this `Http2Stream`.
 
+#### http2stream.trailers(headers)
```

Comment:
Are we OK with using non-verbs for method names? There are some other examples in http2 API, so this is just to be sure that this is intended)

Reply:
maybe this can be `sendTrailers()`?